socorro/processor/signature_utilities.py

```diff
@@ -998,3 +998,35 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
             processed_crash['classifications']['jit']['category']
         )
         return True
+
+
+#==============================================================================
+class SignatureIPCChannelError(Rule):
+    """replaces the signature if there is a JIT classification in the crash"""
+
+    #--------------------------------------------------------------------------
+    def version(self):
+        return '1.0'
+
+    #--------------------------------------------------------------------------
+    def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
+        try:
+            return bool(raw_crash['ipc_channel_error'])
```

Comment:
Because I like that ``predicate`` returns a true boolean value. And I don't like it exposing values. I don't know. It's just me being a bit too much of a purist, I guess. :)